src/main/java/Library.java

```diff
@@ -566,4 +568,26 @@ public static int findLevenshteinDistance(String word1, String word2) {
       return stream.collect(Collectors.toList());
     }
   }
+
+  /**
+   * Convert InputStream to String.
+   *
+   * @param inputStream InputStream to convert
+   * @return String
+   * @throws IOException if an I/O error occurs
+   */
+  public static String inputStreamToString(InputStream inputStream) throws IOException {
+    var stringBuilder = new StringBuilder();
```

Comment:
We can move `stringbuilder` inside the `try` block so it won't be constructed if stream reader construction throws an exception